join_request/ltfschoen.yml

```
@@ -0,0 +1,492 @@
+collective: Potoc
+member:
+  name: Luke Schoen
+  address: 1dsrQjL34njJ4Y8FXGyxeLnmunPZ6XAvid9jSQe9S4pTUh2
+  github: ltfschoen
```

please fill your matrix handle as per https://github.com/super-collective/collective-cli/pull/4#discussion_r1586380064
```suggestion
  address: 1dsrQjL34njJ4Y8FXGyxeLnmunPZ6XAvid9jSQe9S4pTUh2
  matrix:
  github: ltfschoen
```

---

i've added the matrix handle now